### PR TITLE
reduce mobile immersive header to 80vh

### PR DESCRIPTION
**Pull request by HarryFischer**
## What does this change?
Reduces height of immersive header until desktop - was 100vh now 80vh
## Why?
@ajbreuer @Bryony-Szekeres request
## Screenshots

| Before      | After      |
|-------------|------------|
|  image |  image |
|  image |  image |

### CODE REVIEW

File: dotcom-rendering/src/web/components/ImageComponent.tsx [hunk 1]
```diff
@@ -273,7 +276,6 @@ export const ImageComponent = ({
 					isLazy={!isMainMedia}
 					isMainMedia={isMainMedia}
 				/>
-				{starRating && <PositionStarRating rating={starRating} />}
```
Comment (HarryFischer) [inline dotcom-rendering/src/web/components/ImageComponent.tsx:276]: Note: removed review stars from immersive layout. @oliverlloyd has a PR to add them back into the correct place

Comment (jamesgorrie) [reply dotcom-rendering/src/web/components/ImageComponent.tsx:276]: Do you have the link to the PR?

Comment (oliverlloyd) [reply dotcom-rendering/src/web/components/ImageComponent.tsx:276]: I actually ended up putting this work aside. It's not complicated but we don't have a use case for showing stars on immersive pieces at this point so there's no urgency to this

File: dotcom-rendering/src/web/layouts/headers/ImmersiveHeader.tsx [hunk 2]
```diff
@@ -27,13 +27,14 @@ import { Island } from '../../components/Island';
 import { decidePalette } from '../../lib/decidePalette';
 
 const hasMainMediaStyles = css`
-	height: 100vh;
+	height: 80vh;
 	/**
     100vw is normally enough but don't let the content shrink vertically too
```
Comment (oliverlloyd) [inline dotcom-rendering/src/web/layouts/headers/ImmersiveHeader.tsx:32]:
```suggestion
    80vw is normally enough but don't let the content shrink vertically too
```

**Review by oliverlloyd — APPROVED** [review]
So when are you going to become a full time developer?

**Comment by oliverlloyd** [comment]
I made a small change to increase the number of breakpoints we capture this story at so we have regression for how this code lands at different widths

**Pull request merged**